kannon/kube_util.py

```diff
@@ -0,0 +1,72 @@
+import enum
+from time import sleep
+from kubernetes import client
+from datetime import datetime
+import random
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class JobStatus(enum.Enum):
+    RUNNING = 0
+    SUCCEEDED = 1
+    FAILED = 2
+
+
+def create_job(
+    api_instance: client.BatchV1Api, job: client.V1Job, namespace: str
+) -> None:
+    api_response = api_instance.create_namespaced_job(
+        body=job,
+        namespace=namespace,
+    )
+    logger.debug(f"Job created. status={api_response.status}")
+
+
+def wait_job_until_completion(
```

Comment:
Is this function used?

Reply:
Not used, it is a function for debugging.
It is removed in commit: ee35c30